src/services/PromotionService.ts

```diff
@@ -1,6 +1,12 @@
+import {
+  AddressType,
+  GeocodingAddressComponentType,
+  LocationType,
+  Place,
```

Comment:
These types should have matching counterparts in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0b5bfba2994c91a099cd5bcfd984f6c4c39228e5/types/google__maps/index.d.ts